src/main/java/org/fundacionjala/prog101CoffeeMaker/coffeeMaker/Implementation/CoffeeMaker.java

```diff
@@ -27,6 +27,75 @@ public CoffeeMaker() {
         deliveryPipe = new DeliveryPipe();
     }
 
+    /**
+     * Fill the boiler with cups of water
+     */
+    public void fillBoilerWithWater(final int cupsWater) {
+        boiler.setCupsOfWater(cupsWater);
+    }
+
+    /**
+     * Fill the boiler with cups of water
+     */
+    public void fillFilterWithCoffeeGrains() {
+        filterAndReceptacle.putCoffeeGrains();
+    }
+
+    /**
+     * Returns true if the pot is it's place
+     * @return boolean
+     */
+    public boolean obtainsStartButtonState() {
+        return startButton.getButtonPressed();
+    }
+
+    /**
+     * Returns true if the pot is it's place
+     * @return boolean
+     */
+    public void pressStartButton() {
+        startButton.isPressed();
+    }
+
+    /**
+     * Returns true if the pot is it's place
+     * @return boolean
+     */
+    public boolean isPotInPlace() {
+        return pot.getIsInPlace();
+    }
+
+    /**
+     * Sets the new place of the pot
```

Comment:
could you explain me why the coments and the method. I dont understand. pot in new place What is the new place of pot

I thing that the name method can be better removePot

Reply:
Ok i'll explain better that method